src/styles/utils/reset.scss

```diff
@@ -0,0 +1,27 @@
+body,
+h1,
+h2,
+a,
+ul,
+li,
+p,
+span,
+image,
+button {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: inherit;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
I don't know exactly but someone said put it inside "normalize" file, I should ask later)

Reply:
@ulositska @Mykola-Maik yeah, looks like normalizing css